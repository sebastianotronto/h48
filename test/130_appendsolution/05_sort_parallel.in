L R' B2 F

0
1
rotation UF
