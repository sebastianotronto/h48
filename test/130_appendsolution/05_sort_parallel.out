R' L F B2
Number of solutions: 1
Shortest solution length: 4
Used bytes: 10
